tests/pom.xml

```diff
@@ -195,6 +201,27 @@
     <jaeger.query.port>18080</jaeger.query.port>
 
     <otel-collector.endpoint></otel-collector.endpoint>
+    
+    <!-- Fixed ports used when starting hono services/adapters locally or with fixed ports -->
+    <fixed.auth.amqps.port>5673</fixed.auth.amqps.port>
```

Comment:
as mentioned above, I would rather not rely on fixed ports and host names if at all possible ...

Reply:
Actually it seems it works. Of course if you ports are free. However, if any of used ports are occupied you have option to change that port via properties configuration.